docs/products/cassandra/get-started.md

```diff
@@ -4,34 +4,412 @@ sidebar_label: Get started
+
+   ```bash
+   # Sample command for connecting with the cqlsh client
+
+   ./cqlsh --ssl -u USER_NAME -p PASSWORD HOST_NAME PORT_NUMBER
+   ```
+
+</TabItem>
+<TabItem value="2" label="Terraform">
+Access your new service with the Apache Cassandra client using the environment variables
+assigned to Terraform outputs:
 
 ```bash
-avn service create demo-cassandra       \
-   --service-type cassandra             \
-   --cloud CLOUD_AND_REGION             \
-   --plan PLAN                          \
-   --project PROJECT_NAME
+./cqlsh --ssl --host=$CASSANDRA_HOST --port=$CASSANDRA_PORT --user=$CASSANDRA_USER --password=$CASSANDRA_PASSWORD
 ```
 
-:::note
+</TabItem>
+<TabItem value="3" label="cqlsh client">
+[Connect to your new service with CLI](/docs/products/cassandra/howto/connect-cqlsh-cli)
+using the
+[the `cqlsh` client](https://cassandra.apache.org/doc/stable/cassandra/tools/cqlsh.html).
+</TabItem>
+</Tabs>
 
-- To preview the full list of default flags, run `avn service create -h`.
-- To list the type-specific options, run `avn service types -v`.
+:::tip
+Discover more tools for connecting to Aiven for Apache Cassandra in
+[Connect to Aiven for Apache Cassandra®](/docs/products/clickhouse/howto/list-connect-to-service).
+:::
+
+## Create keyspaces and tables
+
+Use `cqlsh` to create a keyspace and a table where you can insert your data next.
+
+1. Create a keyspace, for example `library`:
+
+   ```sql
+   CREATE KEYSPACE IF NOT EXISTS library WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : '3' };
+   ```
+
+1. Create a table within the `library` keyspace, for example `book_tracker`:
+
+   ```sql
+   CREATE TABLE IF NOT EXISTS library.book_tracker (book_id text PRIMARY KEY, item_count int, genre text, status text, last_update_timestamp timestamp);
+   ```
+
+## Insert data
+
+Use `cqlsh` to write data in the `book_tracker` table, for example:
+
+```sql
+INSERT INTO library.book_tracker (book_id, item_count, genre, status, last_update_timestamp) VALUES ('01234', 6, 'true_crime', 'available', toTimeStamp(now()));
+INSERT INTO library.book_tracker (book_id, item_count, genre, status, last_update_timestamp) VALUES ('56789', 13, 'romance', 'unavailable', toTimeStamp(now()));
+INSERT INTO library.book_tracker (book_id, item_count, genre, status, last_update_timestamp) VALUES ('12345', 55, 'spirituality_religions', 'available', toTimeStamp(now()));
+INSERT INTO library.book_tracker (book_id, item_count, genre, status, last_update_timestamp) VALUES ('67890', 3, 'adventure', 'unavailable', toTimeStamp(now()));
+```
+
+:::tip
+For importing large amount of data or for batch data load, see:
+
+- [Use DSBULK to load, unload, and count data](/docs/products/cassandra/howto/use-dsbulk-with-cassandra)
+- [Migrate to Aiven with the ZDM Proxy](/docs/products/cassandra/howto/zdm-proxy)
+- [Migrate to Aiven with `sstableloader`](https://aiven.io/blog/aiven-now-supports-sstableloader)
```

Comment:
maybe just remove the sstableloader option? the link leads to a blog post, which links to /dev/null ("the complete procedure can be found here" link leads to somewhere that does not exist, probably the old article). sstableloader migration IMHO is legacy, people are not likely to want this, I'd just leave the `dsbulk` and ZDM proxy options